lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -586,7 +586,7 @@ def build_mark_for_capture_xml(money, authorization, parameters = {})
         xml.tag! :Request do
           xml.tag! :MarkForCapture do
             add_xml_credentials(xml)
-            xml.tag! :OrderID, format_order_id(order_id)
+            xml.tag! :OrderID, format_order_id(order_id || parameters[:order_id])
```

Comment:
Apologies for not being more detailed in the issue.

To my knowledge, that behavior is not buggy. However, it only works if you performed the auth using Active Merchant. I'm using Chase's Hosted Pay Form, which performs the auth in an iframe and provides the authorization token. I send the token to the server and capture there, using Active Merchant.

This can be resolved by combining the token and order ID into a string, like Active Merchant does. That's non obvious, though. Given:

1. Chase requires both the authorization token and the order ID to be certified for production;
2. There are two common ways to perform an auth outside of Active Merchant (Hosted Pay Form and Hosted Pay Page);
3. Combining into a string breaks encapsulation by requiring my app to know about Active Merchant's internals (`authorization_string`, which does the combining, is a private method); and
3. This change preserves existing behavior

It seems reasonable to make this a first-class parameter, rather than tacking it on to the authorization.